### ISSUE TITLE
sonarqube-7.9.4-community Web server startup failed: Unable to determine database dialect to use within sonar with dialect null jdbc url

### ISSUE THREAD

**Issue by gitkent** [SonarSource/docker-sonarqube#440]
Please ensure your issue adheres to the following guidelines: 

- [ ] Please check the problem is not already reported, or a known issue documented in [`develop.md`](develop.md)
- [ ] Please include enough details to reproduce the problem: the command executed, the host platform, error messages or relevant logs

Hi all

We are using sonarqube:7.9-community docker image which has new updates pushed couple days ago. Since the new update, our sonar instance was throwing this jdbc error:
```
Web server startup failed: Unable to determine database dialect to use within sonar with dialect null jdbc url 'xxx'
```
and the pod gone into crashlooping. Switching back to 7.9.3-community image tag fixed the issue. 

Note: Our Sonar instance running on Kubernetes Cluster deployed using this [helm-charts](https://github.com/helm/charts/tree/master/stable/sonarqube). We have set `SONARQUBE_JDBC_USERNAME`, `SONARQUBE_JDBC_PASSWORD` and `SONARQUBE_JDBC_URL` environment variable in our deployment.

**Comment by MartijnErmersI** [comment]
We also have this problem.

Since we try to do a clean deploy of the tag 7.9.4-community every day around 6 AM (CEST). We can back up the claim that the tag change is the cause of the issue. We experience the problem since 24-10-2020 6AM.

**Comment by malena-ebert-sonarsource** [comment]
Hi @gitkent, @MartijnErmersI,

which JDBC connector are you using?

**Comment by gitkent** [comment]
Hi @malena-ebert-sonarsource Postgresql for me

**Comment by takicichy** [comment]
@malena-ebert-sonarsource it was broken with this commit: https://github.com/SonarSource/docker-sonarqube/commit/9f23271d61f7ae2eeda0634603d05d10725adf1e#diff-6ddefc57624d4937db2cfcfaf5d1a4cd3425196dc89f8f832014f18b2285a82e 
They add extra single quotes around url

**Comment by MartijnErmersI** [comment]
@malena-ebert-sonarsource We use SQL server. In our docker-compose.yml the following environment variable is set:
  SONARQUBE_JDBC_URL: "jdbc:sqlserver://mssql;databaseName=sonarqube"

**Comment by tobias-trabelsi-sonarsource** [comment]
Hello all,
i am very sorry that this has slipped thru while reviewing the PR that introduced this. I have fixed this error with #441 but this needs to be released to the docker hub as well. This will happen with the Merge of https://github.com/docker-library/official-images/pull/8954
